src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoX509Data.cs

```diff
@@ -60,10 +60,10 @@ public KeyInfoX509Data(X509Certificate cert, X509IncludeOption includeOption)
                     // Can't honor the option if we only have a partial chain.
                     if ((chain.ChainStatus.Length > 0) &&
                         ((chain.ChainStatus[0].Status & X509ChainStatusFlags.PartialChain) == X509ChainStatusFlags.PartialChain))
-                        throw new CryptographicException(CAPI.CERT_E_CHAINING);
+                    throw new CryptographicException(unchecked((int)Interop.Crypt32.CertChainPolicyErrors.CERT_E_CHAINING));
```

Comment:
Well, that's what the code comment says.
`"A certificate chain could not be built to a trusted root authority."` is the message I get back from
`Console.WriteLine(new CryptographicException(unchecked((int)0x800B010A)).Message);`.

So... probably something like Cryptography_Partial_Chain for the resource key.